content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -364,3 +361,92 @@ const result = await prisma.$queryRawUnsafe(query)
 
 console.log(result)
 ```
+
+</tab>
+
+<tab>
+
+<TopBlock>
+
+Prisma Client exposes three methods that allow you to send raw queries to your database:
+
+- Use `$runCommandRaw` to run a command against the database
```

Comment:
```suggestion
- Use `prisma.$runCommandRaw` to run a command against the database
```
so it's in line with the other following example

Maybe a link to MongoDB docs here would help figuring out what is a command here